product_variant_sale_price/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "version": "17.0.1.0.0",
     "category": "Product Management",
     "website": "https://github.com/OCA/product-variant",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "author": "Tecnativa, Rooteam, " "Odoo Community Association (OCA)",
```

Comment:
This change doesn't qualify for co-authorship, so remove it.

Reply:
Do I get automatically added as a contributor, or do I have to manually update the README?

Reply:
The commit attribution is already in GitHub, but for being explicitly in the README file, you have to add yourself in https://github.com/OCA/product-variant/blob/17.0/product_variant_sale_price/readme/CONTRIBUTORS.md and execute `pre-commit`

Reply:
Do I merge as is for this time?